src/Data/Traversable/Linear.hs

```diff
@@ -0,0 +1,76 @@
+{-# LANGUAGE DerivingVia #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE LinearTypes #-}
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE TupleSections #-}
+{-# LANGUAGE TypeOperators #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Data.Traversable.Linear
```

Comment:
I'll make you work a bit more. I realise you pulled this out of the `Functor.Linear` module because of the dependency cycle.

But the correct answer is to make a `Functor.Linear.Internal` module, which is reexported in `Functor.Linear` (only the `Functor.Linear` module should have an export list, `Functor.Linear.Internal` should just expose everything).

That way you can also make a `Data.Functor.Linear.Internal.Traversable` functor, which is later in the dependency chain, and reexport it from `Data.Functor.Linear` and you avoid the multiplication of things to import.

That gives you a bit more work, sorry about that, but it's for a good cause.

Reply:
Sounds good, should be done!